### DISCUSSION TITLE
Is volt theme still alive?

### DISCUSSION THREAD

**Discussion by akutovoy** [primefaces/community#629] [PrimeFaces] [answered]
I am trying to upgrade an ancient JSF app that was built using PrimeFaces 6.x. It also used volt-theme 2.1 theme/layout library.
I did not have much trouble upgrading PrimeFaces to 13.x, but cannot find any links to the latest volt jar. I know that it was a "premium" artifact and it was purchased long time ago. Does this library still exist and if so, where can I find it? 
Greatly appreciate any help!
Alex Kutovoy

**Comment by melloware** [comment, marked as answer]
No it looks like it was retired and was considered a Legacy theme according to the old forums: https://forum.primefaces.org/viewforum.php?f=107&sid=824739480dba593310a522fb0063c86f
